src/pages/Game/Game.js

```diff
@@ -161,11 +157,39 @@ class Game extends Component {
     this.setState({ game, currentHandSelection });
   };
 
+  swapCardSelectAction = cardIndex => {
+    console.log("clicked : " + cardIndex);
+    const game = { ...this.state.game };
+    game.local.hand[cardIndex].selected = !game.local.hand[cardIndex].selected;
+    this.setState({ game });
+  };
+
+  handleSwapCards = () => {
+    this.setState({ swapDialogOpened: false });
+    const game = { ...this.state.game };
```

Comment:
je ferais plutôt `const { game } = this.state;` Après je sais pas si il y a un différence de performance avec ce que tu fais (une copie). C'est probablement pas relevant et c'est peut-être mieux de le laisser comme ça puisque c'est constant avec le reste du code.

Reply:
you're the one who suggested that syntax originally... https://github.com/ay-project/ay-project-frontend-react/pull/5#discussion_r266243610